c-sharp.md

```diff
@@ -77,39 +44,44 @@ To compile this, run the .NET 7 command line:
 dotnet build
 ```
 
-This produces a `ConsoleApp.wasm` file in the `bin/Debug/net7.0` directory. We can execute that with Wasmtime:
-
-```
-$ wasmtime ./bin/Debug/net7.0/ConsoleApp.wasm
-Content-Type: text/plain
-
-Hello, World
-```
+This produces a `SpinPage` file in the `bin/Debug/net7.0` directory. 
 
-To run this using Wagi, we need to create a simple `modules.toml`:
+To run this in Spin, create a `spin.toml` file like this:
 
 ```toml
-[[module]]
+spin_version = "1"
+name = "spin-test"
+trigger = { type = "http", base = "/" }
+version = "1.0.0"
+
+[[component]]
+id = "spin-page"
+source = "bin/Debug/net7.0/SpinPage.wasm"
+[component.trigger]
 route = "/"
-module = "bin/Debug/net7.0/ConsoleApp.wasm"
+executor = { type = "wagi" }
 ```
 
-Then we can run Wagi:
-
-```console
-$ wagi -c modules.toml
-```
+You can now start a server with `spin up`.
 
-And using Curl or a web browser, we can hit `http://localhost:3000` and see the output:
+Using `curl`` or a web browser, we can hit `http://localhost:3000` and see the output:
 
 ```console
 $ curl localhost:3000
 Hello, World
 ```
 
+To build more advanced applications, use the [Spin SDK for .NET](https://github.com/fermyon/spin-dotnet-sdk#fast-startup-using-wizer). This [blog post walks you through a new Spin .NET app](https://www.fermyon.com/blog/webassembly-for-dotnet-developers-spin-sdk-intro)
+
 ## Learn More
 
 Here are some great resources:
-
+- The best way to build Spin .NET apps is with the [Spin .NET SDK](https://github.com/fermyon/spin-dotnet-sdk#fast-startup-using-wizer)
```

Comment:
Filip's blog seems like a good and concrete one for this, and pretty recent https://www.strathweb.com/2023/09/dotnet-wasi-applications-in-net-8-0/